<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>402</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="port_setting_groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>191</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>串口设置</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>151</width>
       <height>172</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="port_name_label">
          <property name="text">
           <string>串口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="port_name_comboBox">
          <item>
           <property name="text">
            <string>COM1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM2</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="baud_rate_label">
          <property name="text">
           <string>波特率</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="baud_rate_comboBox">
          <item>
           <property name="text">
            <string>2400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>38400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>57600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>115200</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="data_bit_label">
          <property name="text">
           <string>数据位</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="data_bit_comboBox">
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="parity_bit_label">
          <property name="text">
           <string>校验位</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="parity_bit_comboBox">
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Even</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Odd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mark</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Space</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="stop_bit_label">
          <property name="text">
           <string>停止位</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="stop_bit_comboBox">
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="por_receive_groupBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>291</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>串口接收的数据</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextBrowser" name="port_receive_textBrowser"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="clear_text_pushButton">
         <property name="text">
          <string>清空数据</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="send_port_groupBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>发送到串口的数据</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>271</width>
       <height>33</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLineEdit" name="send_port_lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="send_data_pushButton">
        <property name="text">
         <string>发送数据</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="open_close_groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>通用设置</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>168</width>
       <height>56</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="open_port_pushButton">
          <property name="text">
           <string>打开串口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="close_port_pushButton">
          <property name="text">
           <string>关闭串口</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="quit_pushButton">
        <property name="text">
         <string>退出程序</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="statusBarMes">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>360</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清华大学·轨道交通控制技术研究中心</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QToolButton" name="open_port_toolButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;打开串口&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>打开串口</string>
       </property>
       <property name="icon">
        <iconset resource="source.qrc">
         <normaloff>:/new/toolBar_PNG/image/open_port.png</normaloff>:/new/toolBar_PNG/image/open_port.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="close_port_toolButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;关闭串口&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>关闭串口</string>
       </property>
       <property name="icon">
        <iconset resource="source.qrc">
         <normaloff>:/new/toolBar_PNG/image/close_port.png</normaloff>:/new/toolBar_PNG/image/close_port.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="send_data_toolButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;发送数据&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>发送数据</string>
       </property>
       <property name="icon">
        <iconset resource="source.qrc">
         <normaloff>:/new/toolBar_PNG/image/send_data.png</normaloff>:/new/toolBar_PNG/image/send_data.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="clear_data_toolButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;清空数据&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>清空数据</string>
       </property>
       <property name="icon">
        <iconset resource="source.qrc">
         <normaloff>:/new/toolBar_PNG/image/clear_data.png</normaloff>:/new/toolBar_PNG/image/clear_data.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="quit_toolButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;退出程序&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>退出程序</string>
       </property>
       <property name="icon">
        <iconset resource="source.qrc">
         <normaloff>:/new/toolBar_PNG/image/quit.png</normaloff>:/new/toolBar_PNG/image/quit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>quit_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quit_toolButton</sender>
   <signal>clicked()</signal>
   <receiver>quit_pushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open_port_toolButton</sender>
   <signal>clicked()</signal>
   <receiver>open_port_pushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>14</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>37</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>close_port_toolButton</sender>
   <signal>clicked()</signal>
   <receiver>close_port_pushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>send_data_toolButton</sender>
   <signal>clicked()</signal>
   <receiver>send_data_pushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear_data_toolButton</sender>
   <signal>clicked()</signal>
   <receiver>clear_text_pushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
